cmd/docker-app/inspect.go

```diff
@@ -15,21 +16,47 @@ var (
 	inspectEnv            []string
 )
 
+// inspectCmd represents the inspect command
```

Comment:
nit: I would remove this comment, since `<name>Cmd` is a pattern in the project

Reply:
FWIW I agree